backend/bitboxbase/handlers/handlers.go

```diff
@@ -77,3 +127,15 @@ func (handlers *Handlers) postConnectElectrumHandler(r *http.Request) (interface
 	}
 	return map[string]interface{}{"success": true}, nil
 }
+
+func (handlers *Handlers) postSyncOptionHandler(r *http.Request) (interface{}, error) {
+	payload := struct {
+		Option string `json:"option"`
+	}{}
+	if err := json.NewDecoder(r.Body).Decode(&payload); err != nil {
+		return nil, err
+	}
+
+	success, err := handlers.base.SyncOption(payload.Option)
+	return map[string]interface{}{"success": success}, err
```

Comment:
don't return `err`, as it prevents sending the real response to the app. See https://github.com/digitalbitbox/bitbox-wallet-app/blob/9b7eebccf89434cc7b20c64e83b4bdb41940d75a/backend/handlers/handlers.go#L727-L732

Note that in the compiled app, no response will be relayed at all in this case, not even the one you can see in the above lines.

Ideally, never return an error from handler functions. We need to decide at some point if we wanna get rid of the error response completely, or reserve it for unexpected errors that just make a log entry and are otherwise silently ignored.